src/Tizen.Account.OAuth2/Tizen.Account.OAuth2/AuthorizationResponse.cs

```diff
@@ -46,6 +46,7 @@ internal AuthorizationResponse(IntPtr handle)
         /// The authroization code.
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
+        [Obsolete("This function is deprecated")]
```

Comment:
Obsolete attribute already means that "This method is deprecated". You don't need to write this message here.
It is necessary to present more detailed reasons or alternatives.